keyboards/xd75/keymaps/hybrid/keymap.c

```diff
@@ -0,0 +1,481 @@
+/* Copyright 2020 Hybrid
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Layer shorthand
+#define _MAIN 0
+#define _LY1 1
+#define _LY2 2
```

Comment:
I would prefer to see these as enums, as they will be easier to manage, long term.

```suggestion
enum layers {
    _MAIN,
    _LY1,
    _LY2,
};
```

Reply:
Fixed.